cmd/verify.go

```diff
@@ -35,13 +37,14 @@ import (
 var Version = "1.0.0"
 
 func init() {
-	allChecks = chartverifier.DefaultRegistry().AllChecks()
+	registryChecks := chartverifier.DefaultRegistry().AllChecks()
+	allChecks = profiles.GetProfile().FilterChecks(registryChecks)
```

Comment:
What about filtering the checks in the command runtime instead of package initialization time?

This also gives the opportunity to extend the `verify` command with a `profile` flag, serving as input for the `profiles.GetProfile()`.

Reply:
Good point. Moved it into the BuildChecks function but this had a roll on effect of the tests which call BuildChecks because the registry passed to it is filtered based on the profile.